magma_smaht/wrangler_utils.py

```diff
@@ -125,3 +125,20 @@ def reset_all_failed_mwfrs(smaht_key: dict):
 def print_error_and_exit(error):
     print(error)
     exit()
+
+
+def set_property(
+    uuid: str,
+    prop_key: str,
+    prop_value: Union[str,int],
```

Comment:
prop_value could also be a list. `Any` is probably fine here. Validation will be done portal side anyway and it will fail if we try to patch some unexpected.